File: HotelWeb/src/app/models/rezervacija.ts
export interface Rezervacija {
  datumKreiranja: Date;
  id: string;   
  datumOd: Date;
  datumDo: Date;
  napomena: string;
  sobaId: string;
}
export interface UpsertRezervacija {
  datumOd: Date;
  datumDo: Date;
  napomena: string;
  korisnikId: number | null;
  sobaId: string;
}

File: HotelAPI/Models/Entities/Rezervacija.cs
namespace HotelAPI.Models.Entities
{
    public class Rezervacija : Entity
    {
        public required DateTime DatumKreiranja { get; set; }
        public required DateTime DatumOd { get; set; }
        public required DateTime DatumDo { get; set; }
        public string? Napomena { get; set; }
        public required int KorisnikId { get; set; }
        public required Korisnik Korisnik { get; set; }
        public required Guid SobaId { get; set; }
        public required Soba Soba { get; set; }

    }
}
